### ISSUE TITLE
Application builded how to use the applcation in Tomcat 

### ISSUE THREAD

**Issue by rmarzocchi84** [geosolutions-it/GeoCollect#1]
I builded the applicatuon following the instructions in the wiki now I have three war: 

- geostore.war
- opensdi2-manager.war
- mapstore.war

I try to copy the war files in may CATALINA_BASE directory (a geoserver install approach) but http://localhost:8080/mapstore give me a 500 error and http://localhost:8080/geostore a 404 error...

How can I put the backend and frontend applications of Geocollect in Tomcat?

**Comment by Gnafu** [comment]
Hi @rmarzocchi84 , 
you should put mapstore.war and geostore.war in the /webapps folder of your tomcat instance.
Tomcat will deploy them and you can follow the deploy in the catalina.out log file

Do you have any log file you can share? Even in private if you a concerned about privacy.

**Comment by rmarzocchi84** [comment]
I add also the opensdi2-manager.war

In the catalina.out this is the only ERROR that I found:

2962 [localhost-startStop-1] ERROR org.springframework.web.context.ContextLoader  - Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'baseConfig' defined in URL [jar:file:/var/lib/tomcat8/webapps/opensdi2-manager/WEB-INF/lib/opensdi2-core-1.1-SNAPSHOT.jar!/applicationContext.xml]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: The config dir from the path '/opt/opensdi_tmp' cannot been read or write or it is not a directory... the Application cannot be started.


Can I try to add only mastore and geostore war files, removing opensdi2-manager.war?

**Comment by Gnafu** [comment]
Yes, you should run GeoStore and MapStore in a Tomcat instance and [OpenSDI](https://github.com/geosolutions-it/OpenSDI-Manager2/tree/64cf896a6686ddafa329ae05feb11981a0c08bc7) in separate one (hence a separate port).
The OpenSDI tomcat should also be started with the parameter  
-DOSDI_CONFIG_DIR=/opt/opensdi_tmp  
where /opt/opensdi_tmp can be any folder you want, as long the tomcat user can write into it.

I'll update the documentation to make it more clear.

**Comment by rmarzocchi84** [comment]
OK! Thanks. Let me know if I can help you
